pkg/summary/cattletypes.go

```diff
@@ -34,6 +34,7 @@ func checkRelease(obj data.Object, _ []Condition, summary Summary) Summary {
 	for _, resources := range obj.Slice("spec", "resources") {
 		summary.Relationships = append(summary.Relationships, Relationship{
 			Name:       resources.String("name"),
+			Namespace:  resources.String("namespace"),
```

Comment:
Any chance we can add a test with a resource that's namespaced-scope and a resource that's cluster-scoped?

Reply:
@tomleb No problem, added some tests for `checkRelease`

Reply:
Thanks for those test :rocket: